### DISCUSSION TITLE
Replace images with icons?

### DISCUSSION THREAD

**Discussion by bacoords** [WordPress/wp-feature-notifications#105] [User Interface]
The on-page notifications images have been a topic of conversation- are they even necessary? What about using the smaller, inline 'icons' that we see in the sidebar hub?

https://make.wordpress.org/core/2022/07/21/request-for-feedback-feature-notifications-proof-of-concept/#comment-43432
https://make.wordpress.org/core/2022/07/21/request-for-feedback-feature-notifications-proof-of-concept/#comment-43436

**Comment by erikyo** [comment]
Instead, I was wondering if we can make that the size of the notifications changes depending on the width of the screen...these that we have in the demo now are okay for a very large monitor but in a small monitor (as they pointed out to us) are really important!

What I would propose is a dynamic padding resize (I am referring to the dashboard notifications) in order to mitigate the issue on smaller monitors, something like `clamp(1rem, 8vw, 2rem)`
